src/cli/ui.c

```diff
@@ -0,0 +1,256 @@
+
+// TODO: Convert to thread
+static inline void netdata_cli_run_specific_command(wchar_t *cmd, BOOL root, BOOL checkRet)
+{
+    wchar_t localPath[MAX_PATH + 1] = { };
+    DWORD length = GetCurrentDirectoryW(MAX_PATH, localPath);
+    if (!length) {
+        MessageBoxW(NULL, L"Cannot find binary.", L"Error", MB_OK|MB_ICONERROR);
+        return;
+    }
+    if (root) {
+        // Remove usr\bin
+        length -= 7;
+    }
+    wcscpy(&localPath[length], cmd);
+
+    STARTUPINFO si;
+    PROCESS_INFORMATION pi;
+
+    ZeroMemory( &si, sizeof(si) );
+    si.cb = sizeof(si);
+    ZeroMemory( &pi, sizeof(pi) );
+
+    if(!CreateProcess(NULL, localPath, NULL, NULL, FALSE, 0, NULL, NULL, &si, &pi )) {
+        MessageBoxW(NULL, L"Cannot start process.", L"Error", MB_OK|MB_ICONERROR);
+        return;
+    }
+
+    if (checkRet) {
+        WaitForSingleObject(pi.hProcess, INFINITE);
+
+        CloseHandle( pi.hProcess );
+        CloseHandle( pi.hThread );
+
+        DWORD ret;
+        GetExitCodeProcess(pi.hProcess, &ret);
+
+        wchar_t *msg;
+        wchar_t *title;
+        UINT flags;
+
+        if (ret) {
+            msg = L"Netdata returned error, check your logs.";
+            title = L"Error";
+            flags = MB_OK|MB_ICONERROR;
+        } else {
+            msg = L"Netdata ran command with success!";
+            title = L"Success";
+            flags = MB_OK|MB_ICONINFORMATION;
+        }
+
+        MessageBoxW(NULL, msg, title, flags);
+    }
+}
+
+static LRESULT CALLBACK NetdataCliProc(HWND hNetdatawnd, UINT message, WPARAM wParam, LPARAM lParam)
+{
+    HWND hwndReloadHealth, hwndReloadLabels, hwndSaveDatabase, hwndReopenLogs, hwndStopService, hwndOpenMsys,
+        hwndExit;
+    static HINSTANCE hShell32 = NULL;
+
+    switch (message) {
+        case WM_CREATE: {
+            hwndReloadHealth = CreateWindowExW(0, L"BUTTON", L"Reload Health",
+                                               WS_CHILD | WS_VISIBLE,
+                                               20, 20, 120, 30,
+                                               hNetdatawnd, (HMENU)IDC_RELOAD_HEALTH,
+                                               NULL, NULL);
+
+            hwndReloadLabels = CreateWindowExW(0, L"BUTTON", L"Reload Labels",
+                                               WS_CHILD | WS_VISIBLE,
+                                               20, 60, 120, 30,
+                                               hNetdatawnd, (HMENU)IDC_RELOAD_LABELS,
+                                               NULL, NULL);
+
+            hwndReopenLogs = CreateWindowExW(0, L"BUTTON", L"Reopen Logs",
+                                             WS_CHILD | WS_VISIBLE,
+                                             20, 100, 120, 30,
+                                             hNetdatawnd, (HMENU)IDC_REOPEN_LOGS,
+                                             NULL, NULL);
+
+            hwndSaveDatabase = CreateWindowExW(0, L"BUTTON", L"Save Database",
```

Comment:
This option doesn't even exist.

Reply:
I saw the PR your merged, I am going to simplify this PR later according the answer given [here](https://github.com/netdata/netdata/pull/18485#issuecomment-2344330126) and I am going to remove some buttons like this.